rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :person, :date, :book
+
+  def initialize(person, book, date)
+    @person = person
+    @book = book
+    @date = date
+    person.rentals.push(self) unless person.rentals.include?(self)
+    book.rentals.push(self) unless person.rentals.include?(self)
```

Comment:
- You have done well so far, but this does not properly establish the `belongs-to` association between the `Rental` class and `Book` class, and between the `Rental` class and `Person` class. Please fix this 🙏. You might find this [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) useful